templates/webapi/main/group_overview.html.ep

```diff
@@ -1,5 +1,5 @@
 % layout 'bootstrap';
-% title '';
+% title ((defined $group->{parent_name} ? $group->{parent_name}  . " / " : "") . $group->{name});
```

Comment:
```suggestion
% title $group->{full_name};
```
And populate that within the controller. (In `lib/OpenQA/WebAPI/Controller/Main.pm`, extend the `$group_hash` variable, e.g. in the subsequent if-block:

```
if (!$is_parent_group && (my $parent = $group->parent)) {
        $group_hash->{parent_id} = $parent->id;
        $group_hash->{parent_name} = $parent->name;
        $group_hash->{full_name} = $group->full_name;
    }
```
)

Reply:
Maybe `parent_group_overview.html.ep` could now use `% title $group->{full_name};` as well